packages/core/admin/shared/contracts/audit-logs.ts

```diff
@@ -0,0 +1,43 @@
+import { errors } from '@strapi/utils';
+import { Entity, SanitizedAdminUser } from './shared';
+import { Schema } from '@strapi/types';
+
+interface CollectionAuditLog extends Schema.CollectionType {
+  user?: SanitizedAdminUser;
+}
+
+interface SingleAuditLog extends Schema.SingleType {
+  user?: SanitizedAdminUser;
+}
```

Comment:
doesnt the audit log also return the payload of the log?

Reply:
Isn't that what the schema is? 😬 or is this not an entity?

Reply:
oops, I didnd see the extends :D